DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7/ToHost/RhinoConverterToHost.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Speckle.Converters.Common;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.Rhino7.ToHost;
+
+public class RhinoConverterToHost : ISpeckleConverterToHost
+{
+  public object Convert(Base target) => throw new NotImplementedException();
```

Comment:
I thought from the PR name there is at least one receive operation/class created but it is a ReceiveBinding implementation? If so I would rename PR

Reply:
I stopped on convert because it validated everything I needed to without going into any more complexity.

Any changes related to conversions are coming in another PR following this one, included raw conversions, to host, etc... to prevent this PR from getting messy.

I'll update the title to reflect this, good point!

Reply:
I think it's fair, possinly be nice if it didn't explode tbh but okay :D